Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR /ng-app
 COPY . .
 
 ## Build the angular app in production mode and store the artifacts in dist folder
-RUN export NODE_OPTIONS=--max_old_space_size=8192 && $(npm bin)/ng build --configuration production --aot --base-href='$ARLAS_WUI_BASE_HREF/'
+RUN export NODE_OPTIONS=--max_old_space_size=8192 && npm run build
```

Comment:
`export NODE_OPTIONS=--max_old_space_size=8192` was remove in hub, is it possible for the wui too ?